proto/penumbra/penumbra/core/component/ibc/v1/ibc.proto

```diff
@@ -54,6 +54,9 @@ message Ics20Withdrawal {
   // Whether to use a "compat" (bech32, non-m) address for the return address in the withdrawal,
   // for compatability with chains that expect to be able to parse the return address as bech32.
   bool use_compat_address = 8;
```

Comment:
we could slap a protobuf deprecation marker on this

Reply:
oh cool